packages/channels/package.json

```diff
@@ -31,6 +31,7 @@
     "cli-color": "^2.0.1",
     "express": "^4.17.2",
     "joi": "^17.6.0",
```

Comment:
I tried using the @vonage/server-sdk packages but it's a total mess. You can only make sms request with it, for some reason they dropped the whatsapp functions between version 2.10.7 and 2.10.10. So instead we just make the http request ourselves